spotify_player/src/state/model.rs

```diff
@@ -155,6 +155,21 @@ pub struct Playlist {
     pub name: String,
     pub owner: (String, UserId<'static>),
     pub desc: String,
+    #[serde(default)]
+    pub is_folder: bool,
+    #[serde(default)]
+    pub level: (i32, i32), // current + target
```

Comment:
I don't fully understand this pair. What are current and target?

Reply:
My bad, I should have explained this at the beginning. Will do so later after resolving all comments.

Reply:
Explained in the 'How it works' comment